pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -845,12 +874,12 @@ class DelayedSaturatedMMM(
         import numpy as np
         import pandas as pd
 
-        from pymc_marketing.mmm import DelayedSaturatedMMM
+        from pymc_marketing.mmm import MMM
 
         data_url = "https://raw.githubusercontent.com/pymc-labs/pymc-marketing/main/data/mmm_example.csv"
         data = pd.read_csv(data_url, parse_dates=["date_week"])
 
-        mmm = DelayedSaturatedMMM(
+        mmm = MMM(
             date_column="date_week",
```

Comment:
There aren't defaults
```suggestion
            date_column="date_week",
            adstock="geometric", 
            saturation="logistic", 
```